src/common/localization/en-us.ts

```diff
@@ -691,6 +691,10 @@ export const english: IAppStrings = {
         getOcrError: {
             title: "Cannot load OCR file",
             message: "Failed to load from OCR file. Please check your connection or network settings.",
+        },
+        onPremiseConnectionError: {
+            title: "Cannot access files or service",
+            message: "Failed access files or service. Please check your connection or network settings.",
```

Comment:
1- "Missing the "to" in "Failed **to** access files..."
2- I think we can be more specific here since we know they're using an on premise solution and training failed. Something like: "Failed to access files or service. Please check if your container is configured correctly and is running."